openmp/libomptarget/plugins-nextgen/cuda/src/rtl.cpp

```diff
@@ -373,16 +374,21 @@ struct CUDADeviceTy : public GenericDeviceTy {
     return Plugin::check(Res, "Error in cuCtxSetCurrent: %s");
   }
 
+  /// NVIDIA returns the product of the SM count and the number of warps that
+  /// fit if the maximum number of threads were scheduled on each SM.
+  uint64_t getHardwareParallelism() const override {
+    return HardwareParallelism;
```

Comment:
Where is this value set?

Reply:
This is borrowing it from a previous patch that was added for the RPC support. It's currently set at line 309.